backend/app/controllers/spree/admin/customer_returns_controller.rb

```diff
@@ -31,15 +31,15 @@ def build_resource
       end
 
       def find_resource
-        Spree::CustomerReturn.accessible_by(current_ability, :show).find(params[:id])
+        current_store.customer_returns.accessible_by(current_ability, :show).find(params[:id])
       end
 
       def collection
         parent # trigger loading the order
-        @collection ||= Spree::ReturnItem.
+        @collection ||= current_store.customer_returns.where(id: Spree::ReturnItem.
                         accessible_by(current_ability).
                         where(inventory_unit_id: @order.inventory_units.pluck(:id)).
```

Comment:
```suggestion
                        joins(return_items: { order: :inventory_units }).where(inventory_unit: { return_items: { order: :inventory_units }}).
```
Or something like that could work.